.pylintrc

```diff
@@ -0,0 +1,43 @@
+[MASTER]
+
+# Add files or directories to the blacklist.
+ignore=LICENSE,
+       MANIFEST.in,
+       PUBLISH.md,
+       README.md,
+       requirements.txt,
+       script.test,
+
+
+[MESSAGES CONTROL]
+
+disable=broad-except,
```

Comment:
Can we break this up in two categories just visually.

```
disable=stuff-that-we-actually-want-ignored,
       stuff-that-we-actually-want-ignored2,

       stuff-thats-only-ignored-for-now-and-will-be-addressed-in-a-pr,
       stuff-thats-only-ignored-for-now-and-will-be-addressed-in-a-pr2

```

Reply:
@Razzeee I think mostly all f them are ones which we are ignoring temporarily except import-error and no-name-in-module as It will cause build to fail due to virtual environment (as mentioned in https://github.com/PyCQA/pylint/issues/1524 )